### ISSUE TITLE
Fix Playlist for Video

### ISSUE THREAD

**Issue by GoogleCodeExporter** [peteaky/android-xbmcremote#323]
```
What steps will reproduce the problem?
1. Play a list or folder of video files.
2. Press "Now Playing"
3. Press the Playlist icon at the bottom right of the screen.

What is the expected output? What do you see instead?
Expected output would be a list of videos in the queue. Instead, the screen 
says "Music Playlist" and "no tracks in playlist"


What version of XBMC Remote are you using?
0.8.2-b1


Which Android device are you using (and which ROM, if custom)?
Sprint Hero CDMA. Cyanogen Mod 6.0-rc0 for Sprint Hero.


Which XBMC revision are you running and on which platform?
9.11 on Windows 7


Please provide any additional information below.

```

Original issue reported on code.google.com by `justin.b...@gmail.com` on 20 Jul 2010 at 9:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `phree...@gmail.com` on 20 Jul 2010 at 9:20
- Changed title: **Fix Playlist for Video**
- Changed state: **Accepted**
- Added labels: **Milestone-Release1.0**, **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
I'd like to add that this is an issue for music playlists, as well:
1. Choose "Listen to Music". Navigate to a folder with playlist files (.pls)
2. The playlist files have folder icons. If you tap one, the path will change 
to /whatever/playlistfile.pls, and the heading will say "Not folder"
3. Long-press a playlist file and choose "Play folder" (or "Queue folder"). You 
will see a toast saying "Playing /whatever/playlistfile.pls".

Expected behavior would be that the playlist is loaded and starts to play. 
Actual behavior is that nothing is loaded or played (Now Playing says "No 
tracks in playlist").

Still, many thanks for a first-class remote!
```

Original comment by `matias.w...@gmail.com` on 10 Oct 2010 at 12:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `Gabor.Fo...@gmail.com` on 30 Nov 2010 at 9:51

**Comment by GoogleCodeExporter** [comment]
```
Implemented in v0.8.5 beta1
```

Original comment by `Gabor.Fo...@gmail.com` on 29 Jan 2011 at 11:07
- Changed state: **Fixed**
